packages/bpk-component-map/src/__snapshots__/BpkMap-test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BpkMap should render correctly 1`] = `<div />`;
```

Comment:
These snapshots are pretty useless - how can we test this component better?

Reply:
This is on the todo list:

>  Snapshot tests aren't right. We probably need to use Enzyme to get the map to load. (See this: tomchentw/react-google-maps#410).

My bad as I should have marked this PR as WIP in this aspect (will do now).